src/Common/src/System/Collections/Generic/LowLevelList.cs

```diff
@@ -137,14 +137,14 @@ public int Capacity
                 {
                     if (value > 0)
                     {
-#if TYPE_LOADER_IMPLEMENTATION
                         T[] newItems = new T[value];
+#if TYPE_LOADER_IMPLEMENTATION
```

Comment:
This ifdef is not needed anymore - Array.Copy will work fine even for TYPE_LOADER_IMPLEMENTATION .